parser.go

```diff
@@ -476,7 +478,7 @@ func ParseRule(rule string) (*Rule, error) {
 	if err != nil {
 		return nil, err
 	}
-	r := &Rule{}
+	r := &Rule{Metas:make(map[string]string)}
```

Comment:
This should stay nil if there are no metadata entries. See how this initialization is handled for Tags
if r.Tags == nil {
  make(...)
}
r.Tags[foo] = bar